pkg/token/token.go

```diff
@@ -564,14 +564,23 @@ func (v tokenVerifier) Verify(token string) (*Identity, error) {
 	req.Header.Set(clusterIDHeader, v.clusterID)
 	req.Header.Set("accept", "application/json")
 
+	stsEndpointType := metrics.InvalidSTSEndpoint
+	if parsedURL.Host == "sts.amazonaws.com" {
```

Comment:
do we have global endpoint for non aws partitions?

Reply:
no, for non aws partition there is no separate global endpoint